.github/CODEOWNERS

```diff
@@ -228,6 +228,12 @@
 # PRLabel: %Cognitive - Anomaly Detector
 /sdk/anomalydetector/                                              @conhua @mengaims @juaduan @moreOver0
 
+# PRLabel: %API Center
+/sdk/apicenter/                                                    @alexyaang @azaslonov
```

Comment:
This needs to move down to the management section, around L930.   Please organize to preserve alphabetical ordering.

Reply:
I think I get what you mean? I made a commit here https://github.com/Azure/azure-sdk-for-net/pull/43013/commits/af80885324bea1030fecb2d186f2aded9bd247f5, is this what you meant?